deployments/compose/docker-compose.dev.yml

```diff
@@ -57,7 +57,7 @@ services:
 
   # The CometBFT node
   cometbft-node0:
-    image: "docker.io/cometbft/cometbft:v0.37.9"
+    image: "docker.io/cometbft/cometbft:v0.37.15"
```

Comment:
Frankly I'm in favor of deleting the docker-compose files, because I don't believe they're used anywhere, but they may serve as a reference to someone trying to write their own, so leaving in-tree for now.